File: Programming Basics with JS/T04 - Nested Conditional Statements/exercise/03. New House.js
function newHome(input) {
    let typeOfFlower = input[0];
    let countOfFlowers = Number(input[1]);
    let budget = Number(input[2]);
    let price = 0;


    switch (typeOfFlower) {
        case "Roses": price = countOfFlowers * 5.00;
            if(countOfFlowers > 80){
                price = price * 0.90;
            }
            break;

        case "Dahlias": price = countOfFlowers * 3.80;
            if(countOfFlowers > 90){
                price = price * 0.85;
            }
            break;

        case "Tulips": price = countOfFlowers * 2.80;
            if(countOfFlowers > 80){
                price = price * 0.85;
            }
            break;

        case "Narcissus": price = countOfFlowers * 3.00;
            if(countOfFlowers < 120){
                price = price * 1.15;
            }
            break;

        case "Gladiolus": price = countOfFlowers * 2.50;
            if(countOfFlowers < 80){
                price = price * 1.20;
            }
            break;
    }
    let diff = Math.abs(price - budget);

    if (budget >= price) {
        console.log(`Hey, you have a great garden with ${countOfFlowers} ${typeOfFlower} and ${diff.toFixed(2)} leva left.`)
    } else {
        console.log(`Not enough money, you need ${diff.toFixed(2)} leva more.`)
    }
}
newHome(["Roses", "55", "250000"]);

File: Programming Basics with Python/T04 - Conditional Statements (Advanced)/exercise/03. New House.py
flower_type = input()
quantity = int(input())
budget = int(input())
price_flowers = 0
amount_left = 0
amount_short = 0

if flower_type == "Roses":
    price_flowers = quantity * 5
    if quantity > 80:
        price_flowers -= price_flowers * 0.10

elif flower_type == "Dahlias":
    price_flowers = quantity * 3.80
    if quantity > 90:
        price_flowers -= price_flowers * 0.15

elif flower_type == "Tulips":
    price_flowers = quantity * 2.80
    if quantity > 80:
        price_flowers -= price_flowers * 0.15

elif flower_type == "Narcissus":
    price_flowers = quantity * 3
    if quantity < 120:
        price_flowers += price_flowers * 0.15

elif flower_type == "Gladiolus":
    price_flowers = quantity * 2.50
    if quantity < 80:
        price_flowers += price_flowers * 0.20

if budget >= price_flowers:
    amount_left = budget - price_flowers
    print(f"Hey, you have a great garden with {quantity} {flower_type} and {amount_left:.2f} leva left.")

else:
    amount_short = price_flowers - budget
    print(f"Not enough money, you need {amount_short:.2f} leva more.")